modules/effects/src/main/scala/dev/profunktor/redis4cats/algebra/strings.scala

```diff
@@ -32,6 +32,7 @@ trait Getter[F[_], K, V] {
   def get(key: K): F[Option[V]]
   def getBit(key: K, offset: Long): F[Option[Long]]
   def getRange(key: K, start: Long, end: Long): F[Option[V]]
+  def getDel(key: K): F[Option[V]]
```

Comment:
[KeyCommands](https://github.com/profunktor/redis4cats/blob/series/1.x/modules/effects/src/main/scala/dev/profunktor/redis4cats/algebra/keys.scala#L26) might be a better place for this. We also have `getSet` in `Setters`, so that would be more consistent, I think.

Reply:
``KeyCommands`` doesn't have ``V`` though, which is required for ``getDel`` to return the deleted value

Reply:
Fair enough, I guess it's fine here then 👍🏾

Reply:
Alright sounds good! I'll add the test case sometime tomorrow

Reply:
Awesome, thanks!